docker-compose.connect.yml

```diff
@@ -1,27 +1,33 @@
 version: '2.1'
 services:
-  #
-  # mender-deviceconnect
-  #
-  mender-deviceconnect:
-    image: mendersoftware/deviceconnect:mender-master
-    command: server --automigrate
-    extends:
-      file: common.yml
-      service: mender-base
-    networks:
-      - mender
-    depends_on:
-      - mender-mongo
-    environment:
-      DEVICECONNECT_MONGO_URL: "mongodb://mender-mongo"
-      DEVICECONNECT_NATS_URL: "nats://mender-nats"
 
-  mender-nats:
-    image: nats:2.1-alpine3.11
-    networks:
-      - mender
+    #
+    # mender-deviceconnect
+    #
+    mender-deviceconnect:
+        image: mendersoftware/deviceconnect:mender-master
+        command: server --automigrate
+        extends:
+            file: common.yml
+            service: mender-base
+        networks:
+            - mender
+        depends_on:
+            - mender-mongo
```

Comment:
technically it depends on nets as well.

Reply:
Right: https://github.com/mendersoftware/integration/pull/1110/commits/7922d1162cc0de988beba6353e513761dd1d1c46